src/components/top-bar/styles.ts

```diff
@@ -0,0 +1,87 @@
+import { css } from '@linaria/core'
+import { styled } from '@linaria/react'
+import { isDesktopOrBelow, isMobile, isTabletOrBelow } from '../../styles/media'
+import { ElButtonGroup } from '../button-group'
+import { ElNavSearchButton } from '../nav-search-button/styles'
+import { ElButtonNavIconItem } from '../nav-icon-item'
+
+export const elTopBarAvatar = css`
+  display: flex;
+  padding: var(--spacing-1, 4px) var(--spacing-none, 0px);
+  padding-left: var(--spacing-2, 8px);
+  align-items: center;
+  gap: 10px;
+`
+
+export const ElTopBarMainNav = styled(ElButtonGroup)`
+  display: flex;
+  padding: var(--spacing-1, 4px) var(--spacing-none, 0px) var(--spacing-1, 4px) var(--spacing-6, 24px);
```

Comment:
**note:** I realise you're probably just copying and pasting the CSS that Figma provides for these components, but I think this is another area where we need to apply our own experience and judgement to our code. The designers (and Figma) do not care what the code is like to maintain, but we do. As such, I think its important that we, as engineers, don't just copy-paste what other tools suggest; rather we should think critically about what is needed to achieve the required layout.

In this case, I think all we need is:
```css
display: flex;
flex-direction: row;
align-items: center;
padding-left: var(--spacing-6, 24px);
flex-grow: 1;
```

Using `align-items: center` allows us to avoid the complex padding definition and simplify it down to specifying the `padding-left`.

Reply:
yes, thanks for the clarity👍